iXBRLViewerPlugin/iXBRLViewer.py

```diff
@@ -207,6 +207,7 @@ def createViewer(self, scriptUrl="js/dist/ixbrlviewer.js"):
         self.roleMap.getPrefix(XbrlConst.documentationLabel, "doc")
         self.roleMap.getPrefix(XbrlConst.summationItem, "calc")
         self.roleMap.getPrefix(XbrlConst.parentChild, "pres")
+        self.roleMap.getPrefix(XbrlConst.dimensionDefault, "d-d")
```

Comment:
@paulwarren-wk Are the javascript changes here compatible with previously generated?  Perhaps try swapping the generated .js into an existing viewer.  @derekgengenbacher-wf This should probably be released as 1.1.0.

Reply:
Yes, although I'm not sure if we should handle it as we currently do, which is to display an "outline unavailable" message:

![image](https://user-images.githubusercontent.com/45077928/142082943-7043f7bf-8012-424f-bae0-5c33b3ac1b8e.png)

Or to hide the outline icon completely if there are no presentation relationships.

Reply:
I don't feel strongly, that looks fine to me.  So long as it doesn't completely break, I think it's ok.  We have only recently advertised support for updating the JS in place for old reports, so I don't know how many users are trying to do that.  If someone is and they don't like it, we can always make a change later once we have a real use case.

Reply:
I discussed this with @davecasleton-wf last week, and we also had no strong feelings, so I agree that we should leave it as is.